docs/Home.md

```diff
@@ -41,7 +41,8 @@ to publish deployments built within Jenkins to your Kubernetes clusters running
     export SA_EMAIL=$SA@$PROJECT.iam.gserviceaccount.com
     gcloud projects add-iam-policy-binding --member serviceAccount:$SA_EMAIL --role roles/iam.serviceAccountUser $PROJECT
     gcloud projects add-iam-policy-binding --member serviceAccount:$SA_EMAIL --role roles/container.clusterAdmin $PROJECT
```

Comment:
We shouldn't need the cluster admin role any longer with this change.

Reply:
Done.